src/main/kotlin/racingcar/game/PrintUtil.kt

```diff
@@ -0,0 +1,48 @@
+package racingcar.game
+
+object PrintUtil {
+    private const val START_MESSAGE = "경주할 자동차 이름을 입력하세요.(이름은 쉼표(,) 기준으로 구분)"
+    private const val ASK_ATTEMPTS_MESSAGE = "시도할 횟수는 몇 회인가요?"
+    private const val EXECUTION_RESULT_MESSAGE = "\n실행 결과"
+    private const val WINNER_MESSAGE = "최종 우승자 : "
+    private const val SOLE_WINNER_COUNT = 1
+
+    fun printStartMessage() {
+        println(START_MESSAGE)
+    }
```

Comment:
[Kotlin 코드 컨벤션](https://kotlinlang.org/docs/coding-conventions.html#functions)에서는 단일 표현식 일때, 아래와 같이 수정하는 것을 권장하고 있습니다! 14-20 라인도 동일합니다
```suggestion
    fun printStartMessage() = println(START_MESSAGE)
```

Reply:
오 출력 부분도 해당하는군요. 좋은리뷰 감사드립니다 ㅎㅎ🫢